### PR TITLE
[IMP] port module sale_automatic_workflow_exception to 8 version

### PR DESCRIPTION
**Pull request by sebastienbeau**

### CODE REVIEW

**Comment by pedrobaeza** [comment]
Can you please respect commit history? You can use a procedure like https://github.com/odoomrp/odoomrp-wip/wiki/Migration-to-OCA

**Comment by pedrobaeza** [comment]
Sorry for the confussion. No need to extract history. I have checked badly.

File: sale_automatic_workflow_exception/README.rst [hunk 1]
```diff
@@ -0,0 +1,78 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+==============
+Sale Automatic workflow exception
+==============
```
Comment (pedrobaeza) [inline sale_automatic_workflow_exception/README.rst]: Please put the same length as the title (both above and below lines)

File: sale_automatic_workflow_exception/README.rst [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+============
+
+This module will be installed automatically if Sale Automatic Workflow and Sale Exception
+are installed
+
+Configuration
+=============
+
+Activate the cron "Test Draft Orders"
+* go to Setting > Technical > Automation > Scheduled Actions
```
Comment (pedrobaeza) [inline sale_automatic_workflow_exception/README.rst]: An extra line is needed here for being seen correctly

File: sale_automatic_workflow_exception/README.rst [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+
+Usage
+=====
+
+
+.. image:: https://odoo-community.org/website/image/ir.attachment/5784_f2813bd/datas
+   :alt: Try me on Runbot
+   :target: https://runbot.odoo-community.org/runbot/167/8.0
+
+Known issues / Roadmap
```
Comment (pedrobaeza) [inline sale_automatic_workflow_exception/README.rst]: Remove this empty section

File: sale_automatic_workflow_exception/__openerp__.py [hunk 4]
```diff
@@ -1,44 +1,22 @@
...
-#
-###############################################################################
+# © 2014 Akretion (http://www.akretion.com)
+# Sébastien BEAU <sebastien.beau@akretion.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
 
 {'name': 'Sale Automatic Workflow Exception',
- 'version': '0.0.1',
+ 'version': '8.0.0.0.1',
```
Comment (pedrobaeza) [inline sale_automatic_workflow_exception/__openerp__.py]: 8.0.1.0.0

File: sale_automatic_workflow_exception/__openerp__.py [hunk 5]
```diff
@@ -1,44 +1,22 @@
...
-#   This program is distributed in the hope that it will be useful,
-#   but WITHOUT ANY WARRANTY; without even the implied warranty of
-#   MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
-#   GNU Affero General Public License for more details.
-#
-#   You should have received a copy of the GNU Affero General Public License
-#   along with this program.  If not, see <http://www.gnu.org/licenses/>.
-#
-###############################################################################
+# © 2014 Akretion (http://www.akretion.com)
```
Comment (pedrobaeza) [inline sale_automatic_workflow_exception/__openerp__.py]: 2014-2016

File: sale_automatic_workflow_exception/automatic_workflow_job.py [hunk 6]
```diff
@@ -1,33 +1,16 @@
...
-from openerp.osv import orm
+from openerp import models
 
 
-class AutomaticWorkflowJob(orm.Model):
+class AutomaticWorkflowJob(models.Model):
     _inherit = 'automatic.workflow.job'
 
-    def _get_domain_for_sale_validation(self, cr, uid, context=None):
+    def _get_domain_for_sale_validation(self):
```
Comment (pedrobaeza) [inline sale_automatic_workflow_exception/automatic_workflow_job.py:13]: Better to make explicit the use of `@api.multi`

**Comment by hparfr** [comment]
:+1:

File: sale_automatic_workflow_exception/__init__.py [hunk 7]
```diff
@@ -1,23 +1,7 @@
...
-#   You should have received a copy of the GNU Affero General Public License
-#   along with this program.  If not, see <http://www.gnu.org/licenses/>.
-#
-###############################################################################
+# © 2014-2016 Akretion (http://www.akretion.com)
+# Sébastien BEAU <sebastien.beau@akretion.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
 
 from . import automatic_workflow_job
```
Comment (bealdav) [inline sale_automatic_workflow_exception/__init__.py:6]: 1 line is sufficient, here

File: sale_automatic_workflow_exception/__openerp__.py [hunk 8]
```diff
@@ -1,44 +1,22 @@
...
- """,
  'depends': [
      'sale_exceptions',
      'sale_automatic_workflow',
  ],
  'data': [
  ],
- 'installable': False,
+ 'installable': True,
  'application': False,
```
Comment (bealdav) [inline sale_automatic_workflow_exception/__openerp__.py:20]: remove application

**Comment by chafique-delli** [comment]
:+1:

File: sale_automatic_workflow_exception/__openerp__.py (hunk 8 above)
Comment (sebastienbeau) [inline sale_automatic_workflow_exception/__openerp__.py:20]: The key is in the OCA template so I prefer to keep it

**Comment by bealdav** [comment]
:+1:

**Pull request merged**